layouts/partials/home/custom.html

```diff
@@ -97,133 +120,124 @@
 </script>
 
 <style>
-/* Override defaults for homepage */
-.prose {
-  max-width: 100%;
-  margin-bottom: 3rem;
+:where(body):has(.homepage-hero) {
+  // this page's components will handle their own inline spacing
+  margin: unset;
+  padding-inline: unset;
+  max-inline-size: unset;
+
+  :where(main) > *, :where(footer) {
+    // the same content container styles are applied to each component, so 
+    // they can be individually overridden as needed (e.g. for full-bleed layouts)
+    width: 100%;
+    max-width: var(--content-max-width-plus-gutters);
+    margin-inline: auto;
+    padding-inline: var(--page-gutter);
+  }
 }
 
-
-/* Homepage hero styles */
 .homepage-hero {
-  height: 100%;
-  width: 100%;
-  min-height: 30rem;
-  max-height: 35rem; /* Fixed max height to prevent jumping */
+  // allow children (specifically the image) to be full-bleed
```

Comment:
I think this comment is safe, but css comments have strand behaviour so superstitious
```suggestion
  /* allow children (specifically the image) to be full-bleed */
```